tests/featureextract__tests.py

```diff
@@ -9,3 +9,26 @@ def teardown():
 
 def test_basic():
     print "I RAN!"
```

Comment:
Get rid of lines 3 through 11.  def setup() def test_basic() were dummy functions when I setup the package.  These can be removed